cmake/Modules/CodeCoverage.cmake

```diff
@@ -219,6 +231,7 @@ function(ADD_CODE_COVERAGE)
     add_custom_target(${Coverage_NAME}
         DEPENDS ${PROJECT_BINARY_DIR}/${Coverage_NAME}_cpp.info
                 ${PROJECT_BINARY_DIR}/${Coverage_NAME}_nosetests_python.xml
+                ${PROJECT_BINARY_DIR}/${Coverage_NAME}_pytests_python.xml
```

Comment:
if package does not have pytest, it will not be listed and not be uploaded to codecov.
https://github.com/GITAI/gitai_ci/pull/251/files#diff-c548e284ba8a62fe36e150fada177adf30ca006accd3bfd35755d191a4e0f407R76-R79

Reply:
@knorth55 
I see. 
Is it OK for a target depends on non-exist file? I understood that `${PROJECT_BINARY_DIR}/${Coverage_NAME}_pytests_python.xml` is not created if there is not pytest.

Reply:
It seems OK...
The same strategy is set for `nosetests`, and many packages does not have `nosetests` test, but it seems working.
Should I avoid it?

Reply:
Hmm, why does the current implementation work?

The following cmake produces error on my laptop.
```cmake
add_custom_target(foo ALL
                  DEPENDS foo.yml OUTPUT foo.xml
                  COMMAND echo hello world > foo.xml)
```

```
$ make
make[2]: *** No rule to make target `foo.xml', needed by `CMakeFiles/foo'.  Stop.
make[1]: *** [CMakeFiles/foo.dir/all] Error 2
make: *** [all] Error 2
```

Reply:
```
add_custom_command(
  OUTPUT ${PROJECT_SOURCE_DIR}/foo.yml
  COMMAND echo "foo"
)

add_custom_target(
  hoge_test ALL
  DEPENDS ${PROJECT_SOURCE_DIR}/foo.yml
  COMMAND echo "Hoge" 
)
```

When we have `add_custom_command(OUTPUT foo.yml)`, we can build the cmake even if  `add_custom_command` does not generate `foo.yml`.

Reply:
Oh, I didn't know that. Could you add a comment at the top of the `add_custom_target` about it can work even if some DEPENDS files are not generated actually.

Reply:
I added the comment!